pkg/lib/operation/project/remote/table/import/operation.go

```diff
@@ -83,7 +83,7 @@ func getLoadOptions(o *Options) []keboola.LoadDataOption {
 	return opts
 }
 
-func ensureBucketExists(ctx context.Context, d dependencies, rb rollback.Builder, id keboola.BucketID) error {
+func EnsureBucketExists(ctx context.Context, d dependencies, rb rollback.Builder, id keboola.BucketID) error {
```

Comment:
Nejsem si úplně jistej jestli je ok si to půjčovat odtud, jestli by to nemělo být spíš někde v `internal/pkg`, ale potřebuje to parametry specifický pro operace, tak by to mohlo být ok?

Reply:
Imho je to v pohodě znovu použít, když se jedná o další operaci